lang/en.yml

```diff
@@ -1,5 +0,0 @@
-tab_text_project_specific: Project Specific
-tab_text_systemwide: System wide
-tab_label_tab_name: Tab Name
```

Comment:
Probably was the location of the locales for an older version of redmine/rails, obsolete now. I merged into config/locales